apps/public-docsite/src/pages/Styles/FabricIconsPage/FabricIconsPage.tsx

```diff
@@ -26,6 +29,11 @@ export const FabricIconsPage: React.FunctionComponent<IStylesPageProps> = props
 };
 
 function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
+  const icons: React.FC<FluentIconsProps>[] = [];
+  // eslint-disable-next-line guard-for-in
+  for (const iconName in ReactIcons) {
```

Comment:
It's safer to use `Object.keys` because that only includes "own properties"
```suggestion
  for (const iconName of Object.keys(ReactIcons)) {
```